### PR TITLE
[Storage] Temporary fix for file trailing dot live test case 

### PR DESCRIPTION
**Pull request by yifanz7**
## Mandatory Checklist

- Please choose the target release of Azure PowerShell. (⚠️**Target release** is a different concept from **API readiness**. Please click below links for details.)
  - [ ] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] No need for a release

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/Storage/LiveTests/Storage.Management/FileTests.ps1 [hunk 1]
```diff
@@ -12,7 +12,8 @@ Invoke-LiveTestScenario -Name "File basics" -Description "Test File basic operat
     $ctx = $account.Context 
     $ctx1 = New-AzStorageContext -StorageAccountName $storageAccountName -StorageAccountKey (Get-AzStorageAccountKey -ResourceGroupName $rgName -Name $storageAccountName)[0].Value
 
-    $objectName1 = "filetest1.txt." 
+    $objectName1 = "filetest1.txt" 
+    # $objectName1 = "filetest1.txt." removing the trailing dot temporarily to fix the case
```
[the remark this answers is not in the document]
Comment (yifanz7) [reply src/Storage/LiveTests/Storage.Management/FileTests.ps1]: Done.

**Comment by yifanz7** [comment]
@Nickcandy Could you please help take a look at the build failure? I'm not quite sure where this failure happens:

PullRequestNumber:26329
---Exception Caught when trying to detect file changes with PS script: System.ArgumentException: String cannot be empty (Parameter 'token')
   at Octokit.Ensure.ArgumentNotNullOrEmptyString(String value, String name) in /home/runner/work/octokit.net/octokit.net/Octokit/Helpers/Ensure.cs:line 35
   at Octokit.Credentials..ctor(String token, AuthenticationType authenticationType) in /home/runner/work/octokit.net/octokit.net/Octokit/Http/Credentials.cs:line 32
   at Octokit.Credentials..ctor(String token) in /home/runner/work/octokit.net/octokit.net/Octokit/Http/Credentials.cs:line 28
   at Microsoft.WindowsAzure.Build.Tasks.FilesChangedTask.Execute() in D:\a\_work\1\s\tools\BuildPackagesTask\Microsoft.Azure.Build.Tasks\FilesChangedTask.cs:line 99
FilesChanged: 0
D:\a\_work\1\s\build.proj(143,5): error MSB4181: The "CIFilterTask" task returned false but did not log an error.

**Comment by Nickcandy** [comment]
Sorry that we have some issues with our pipeline for now. We would fix them in these days.

**Review by Nickcandy — APPROVED** [review]

**Pull request merged**